sourcegraph/sourcegraph
# own: guard resolver endpoints by feature flag and implement NodeByIDFunc for ingested files

follow-up on leftover comments in https://github.com/sourcegraph/sourcegraph/pull/48182

## Test plan

Manually tested:

```gql
mutation add {
  addCodeownersFile(input: {fileContents: "* @leonore", repoName: "github.com/sourcegraph/sourcegraph"}) {
    id  
    repository {
      id 
      name 
      diskSizeBytes
    }
  }
}

query list {
  codeownersIngestedFiles(first: 2) {
    totalCount
    nodes {
      id
    }
  }
}
```
Manual tests made me notice it wasn't clear when an error came from unmarshalling repository ID so added a wrap there!
